src/pages/Proposal.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+	<div
+		class="header_margin"
+		:style="'--p-height: ' + headerHeight + 'px'"
+	></div>
+	<div class="mt flexBox flexBox_center">
+		<form @submit="onSubmit">
+			<label for="url">URL</label>
+			<input id="url" required v-model="formState.url" type="url" />
+			<label for="url_hash">URL Hash</label>
+			<input id="url_hash" required v-model="formState.url_hash" />
+			<label for="proposal_address">Proposal Account Address</label>
+			<input
+				id="proposal_address"
+				required
+				v-model="formState.proposal_address"
+			/>
+			<label for="url">Proposal ID</label>
+			<input id="url" required v-model="formState.proposal_id" />
+			<label for="voting_date">Voting Date</label>
+			<a-range-picker
+				id="voting_date"
+				format="YYYY-MM-DD HH:mm:ss"
+				value-format="YYYY-MM-DD HH:mm:ss"
+				:disabled-date="disabledDate"
+				:disabled-time="disabledRangeTime"
+				showTime
+				v-model:value="formState.vote_date"
+				style="border-color: black"
+			/>
+			<label for="vote_options">Vote</label>
+			<select
+				name="vote_type"
+				id="vote_options"
+				required
+				v-model="formState.vote_type"
+			>
+				<option :value="ProposalType.ALGO">Algo Transfer</option>
+				<option :value="ProposalType.ASA">ASA Transfer</option>
+				<option :value="ProposalType.MESSAGE">Message</option>
+			</select>
+			<div
+				class="flexBox"
+				v-if="
+					formState.vote_type && formState.vote_type !== ProposalType.MESSAGE
+				"
+			>
+				<label for="from"> From</label>
+				<input id="from" v-model="formState.from" required />
+
+				<label for="recipient">Recipient</label>
+				<input id="recipient" v-model="formState.recipient" />
+
+				<label for="amount"> Amount </label>
+				<input id="amount" v-model="formState.amount" />
+			</div>
+			<div class="flexBox" v-if="formState.vote_type === ProposalType.ASA">
+				<label for="asa_id">ASA ID</label>
+				<input id="asa_id" v-model="formState.asaId" />
+			</div>
+			<div class="flexBox" v-if="formState.vote_type === ProposalType.MESSAGE">
+				<label for="message">Message</label>
+				<input id="message" v-model="formState.message" />
+			</div>
+			<button type="submit" class="submit_btn" @click="onSubmit">Submit</button>
+		</form>
```

Comment:
why not using https://www.antdv.com/components/form/ ?

Reply:
I tried using it, but its validation and error handling is not working as expected. I also had a call with @rmeena840 regarding it but we couldn't find any fix. So I switched to basic html, we can discuss about it later if you want.

Reply:
OK, let's leave it for now. In the past I successful used antd form and it worked well

Reply:
I've created a task for it: https://www.pivotaltracker.com/story/show/181590936

Reply:
BTW: currently we don't have error validation either. Anyway - let's migrate to antd from in a new task.